rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import {readFileSync} from 'fs';
+import common from 'rollup-plugin-commonjs';
+import nodeResolve from 'rollup-plugin-node-resolve';
+
+const banner = readFileSync('./banner.js', 'utf-8');
+const dependencies = process.env.DEPS === 'YES';
+
+export default {
+    banner: banner,
+    entry: './index',
+    format: 'cjs',
+    plugins: dependencies ? [
+        nodeResolve(),
+        common()
```

Comment:
Can you end all list items with a comma please? This makes moving items around really easy and I hate that JSON does not allow this.